src/java/org/apache/cassandra/service/accord/AccordSafeCommandStore.java

```diff
@@ -305,6 +305,7 @@ public void setSafeToRead(NavigableMap<Timestamp, Ranges> newSafeToRead)
     @Override
     public void setRangesForEpoch(CommandStores.RangesForEpoch rangesForEpoch)
     {
+        super.setRangesForEpoch(rangesForEpoch);
```

Comment:
I don't think this is correct? We shouldn't set this until we have written it to the journal?

Reply:
dropped this commit